src/main/java/seedu/application/model/Model.java

```diff
@@ -12,10 +12,6 @@
  * The API of the Model component.
  */
 public interface Model {
-    /** {@code Predicate} that always evaluate to true */
-    Predicate<Application> PREDICATE_SHOW_ALL_APPLICATIONS = unused -> true;
```

Comment:
Having two separate classes for Predicates seems a little unnecessary to me. I think it might be better to follow what they have done here and keep the Predicates with Model class.

Reply:
Hmmm yeah you are right. I created separated class because my implementation initially is different 😅 Let me change back to the original implementation.

Thanks for pointing out!